libraries/DataFlash/LogStructure.h

```diff
@@ -480,6 +480,8 @@ struct PACKED log_Current {
     float    battery_voltage;
     float    current_amps;
     float    current_total;
+    int16_t  temperature; // degrees C * 100
```

Comment:
what do you think about just recording the temperature in C in a signed byte?  It's not the size so much as the slightly awkward C*100 scaling.  I'll leave it up to you.